src/ceph-detect-init/ceph_detect_init/debian/__init__.py

```diff
@@ -14,4 +14,7 @@ def choose_init():
             return 'systemd'
         else:
             return 'upstart'
+    if distro.lower() in ('debian'):
+        if codename >= 'jessie':
```

Comment:
the debian version names aren't sorted... i think we need to enumerate the "old" distros and map them to sysvinit, and then default to systemd for others.

i think that is just squeeze and wheezy -> sysvinit.. anything older won't work anyway.